api/src/pcapi/routes/adage_iframe/educational_institution.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+import logging
+
+from pcapi.core.educational import repository as educational_repository
+from pcapi.routes.adage_iframe import blueprint
+from pcapi.routes.adage_iframe.security import adage_jwt_required
+from pcapi.routes.adage_iframe.serialization import educational_institution
+from pcapi.routes.adage_iframe.serialization.adage_authentication import AuthenticatedInformation
+from pcapi.serialization.decorator import spectree_serialize
+
+
+logger = logging.getLogger(__name__)
```

Comment:
détail : On n'en a pas besoin, on peut supprimer : 
```suggestion
logger = logging.getLogger(__name__)
```